extensions/wikia/TemplateDraft/scripts/rightRailModule.js

```diff
@@ -0,0 +1,36 @@
+define(
+	'ext.wikia.templateDraft.rightRailModule',
+	[
+		'jquery',
+		'wikia.window',
+		'wikia.tracker'
+	],
+	function ($, w) {
+		'use strict';
+
+		function bindCloseModuleLink() {
+			$('.templatedraft-module-closelink-link')
+				.on('click', closeModule);
+		}
+
+		function closeModule() {
+			$('.templatedraft-module').hide();
+
+			$.nirvana.sendRequest({
+				controller: 'TemplateDraftController',
+				method: 'markTemplateAsNotInfobox',
+				data: {
+					'pageId': w.wgArticleId
```

Comment:
In MW 1.19, it's generally preferred to require `mw` and use `mw.config.get('wgArticleId');`.